php/map.jinja

```diff
@@ -1,500 +1,3247 @@
+                },
+                'cli': {
+                    'ini': {
+                        'opts': {},
+                        'settings': {},
+                    }
+                },
+                'apache2': {
+                    'ini': {
+                        'opts': {},
+                        'settings': {},
+                    }
+                },
+                'xcache': {
+                    'ini': {
+                        'defaults': {
+                            'xcache-common': {
+                                'extension':  'xcache.so',
+                            },
+                            'xcache.admin': {
+                                'xcache.admin.enable_auth': 'On',
+                            },
+                            'xcache': {
+                                'xcache.shm_scheme':  '"mmap"',
+                                'xcache.size':  '60M',
+                                'xcache.count': '1',
+                                'xcache.slots': '8K',
+                                'xcache.ttl': '0',
+                                'xcache.gc_interval': '0',
+                                'xcache.var_size':  '4M',
+                                'xcache.var_count': '1',
+                                'xcache.var_slots': '8K',
+                                'xcache.var_ttl': '0',
+                                'xcache.var_maxttl':  '0',
+                                'xcache.var_gc_interval': '300',
+                                'xcache.var_namespace_mode':  '0',
+                                'xcache.var_namespace': '""',
+                                'xcache.readonly_protection': 'Off',
+                                'xcache.mmap_path': '"/dev/zero"',
+                                'xcache.coredump_directory':  '""',
+                                'xcache.coredump_type': '0',
+                                'xcache.disable_on_crash':  'Off',
+                                'xcache.experimental':  'Off',
+                                'xcache.cacher':  'On',
+                                'xcache.stat':  'On',
+                                'xcache.optimizer': 'Off',
+                            },
+                            'xcache.coverager': {
+                                'xcache.coverager': 'Off',
+                                'xcache.coverager_autostart': 'On',
+                                'xcache.coveragedump_directory':  '""',
+                            },
+                        },
+                        'opts': {},
+                        'settings': {},
+                    }
+                },
+                'ini': {
+                    'defaults': {
+                        'PHP': {
+                            'allow_url_fopen': 'On',
+                            'allow_url_include': 'Off',
+                            'asp_tags': 'Off',
+                            'auto_globals_jit': 'On
```

Comment:
```suggestion
                            'xml': ['php' + php_version + '-xml', 'php' + php_version + '-xmlrpc'],
```

Reply:
@sticky-note GitHub seems to be struggling here -- I've clearly indicated line 2267 but it's showing line 1266!  Hopefully, it's clear to you what I'm indicating.